camayoc/tests/qcs/cli/test_credentials.py

```diff
@@ -611,6 +612,102 @@ def test_clear(isolated_filesystem, qpc_server_config):
     qpc_cred_show.close()
 
 
+def test_clear_with_source(isolated_filesystem, qpc_server_config):
+    """Clear an auth which is used in a source.
+
+    :id: 66d84e9c-3124-11e8-b467-0ed5f89f718b
+    :description: Clear one auth entry by entering the ``--name`` of an already
```

Comment:
I think the description could be expanded to match what the test is doing. It is:

1) Creating a credential entry
2) Creating a source and associating with the credential entry
3) Trying to delete the credential
4) Deleting the associated source
5) Deleting the credential